sktime/forecasting/croston.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._sktime import _SktimeForecaster
+from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
+
+
+DEFAULT_SMOOTHING = 0.05
+DEFAULT_ALPHA = 0.05
+
+
+class Croston(_OptionalForecastingHorizonMixin, _SktimeForecaster):
+    """
+    Implementation of Croston's Method
```

Comment:
I think the docstring needs to be updated. See info on [NumPy docstrings](https://numpydoc.readthedocs.io/en/latest/format.html). 

Instead of "Implementation of Croston's Method" followed by underscrores, that probably needs to be a short one line summary.